src/zcbenchmarks.cpp

```diff
@@ -88,3 +90,15 @@ double benchmark_create_joinsplit()
     assert(pourtx.Verify(*pzerocashParams));
     return ret;
 }
+
+double benchmark_solve_equihash()
+{
+    const char *testing = "testing";
```

Comment:
Some clarifications:

* my original code didn't "mine a block." That was a poor choice of words on my part, it just got a solution which may not have met Bitcoin's difficulty.
* Our implementation of equihash might be buggy, because we apparently cannot depend on the solution rate in our parameter choices.